addons/website_slides/models/slide_channel.py

```diff
@@ -177,6 +177,8 @@ def _default_access_token(self):
     can_review = fields.Boolean('Can Review', compute='_compute_action_rights', compute_sudo=False)
     can_comment = fields.Boolean('Can Comment', compute='_compute_action_rights', compute_sudo=False)
     can_vote = fields.Boolean('Can Vote', compute='_compute_action_rights', compute_sudo=False)
+    review_count = fields.Integer('Review count', compute='_compute_review_count', compute_sudo=True)
```

Comment:
I think we could move it to rating_mixin.py and name it rating_review_count . Purpose is

  * to allow its use in other addons by adding it in the mixin. That way all models inheriting from rating mixin will be able to use it if necessary;
  * to keep a well formed namespace (rating_something) for fields, allowing to understand it comes from the rating mixin;